modules/schema_validation.lua

```diff
@@ -169,6 +189,7 @@ local function table_contains(table, element)
 end
 
 
+-- @function Validator:CompareType
```

Comment:
@pestOO you marked this one as function. DO others are not functions? What about parameters for this one?